From 72f7b74c3a798a97cadab30428057e4b5cd60c53 Mon Sep 17 00:00:00 2001
From: Jaikrishna Nemallapudi <nemallapudi.jaikrishna@intel.com>
Date: Mon, 11 Oct 2021 18:51:59 +0530
Subject: [PATCH] Add fix for sensor meta data events.

This patch is added for gettign accelerometer data.

Tracked-On: OAM-99680
Change-Id: I6637d1e7fc967101592585610b8ec1b48abf18f0
Signed-off-by: Jaikrishna Nemallapudi <nemallapudi.jaikrishna@intel.com>
---
 services/sensorservice/SensorEventConnection.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/services/sensorservice/SensorEventConnection.cpp b/services/sensorservice/SensorEventConnection.cpp
index 3cccaf9329..32a5d1a8da 100644
--- a/services/sensorservice/SensorEventConnection.cpp
+++ b/services/sensorservice/SensorEventConnection.cpp
@@ -329,6 +329,7 @@ status_t SensorService::SensorEventConnection::sendEvents(
             // If there is a pending flush complete event for this sensor on this connection,
             // ignore the event and proceed to the next.
             if (flushInfo.mFirstFlushPending) {
+                flushInfo.mFirstFlushPending = false;
                 ++i;
                 continue;
             }
-- 
2.17.1

